helium_hotspot_exporter.py

```diff
@@ -196,8 +214,10 @@ def stats_for_hotspot(addr, hname):
   HOTSPOT_ONLINE.labels(addr,hname).set(isup)
 
   haz_addr = 0
-  if len(d['status']['listen_addrs']):
+  if d['status']['listen_addrs'] and len(d['status']['listen_addrs']):
```

Comment:
what error does this fix? I mean, what value is in there? None?

Reply:
yes